style.css

```diff
@@ -0,0 +1,71 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@700&family=Roboto:ital,wght@0,100;1,100&display=swap");
+
+@media only screen and (min-width: 375px) {
+  .container {
+    width: 100%;
+    height: 718px;
+    font-family: "Poppins", sans-serif;
+  }
```

Comment:
- [ ] Kindly add the background image below to your section to match the figma design. 
![image](https://user-images.githubusercontent.com/91486791/235248617-ba3651a4-e06c-4e84-a7d8-fc2053b72668.png)
the image:
![image](https://user-images.githubusercontent.com/91486791/235248359-ecdb4158-c8a4-43c4-8936-d630f9715e10.png)
- [ ] Please reduce your font weight for the paragraph as you can see in the figma design clearly: 
![image](https://user-images.githubusercontent.com/91486791/235248974-fb944fa7-48af-49a8-a548-ec96468bcb22.png)